TESTS/mbed_hal/spm/main.cpp

```diff
@@ -27,8 +27,8 @@
 #error [NOT_SUPPORTED] Disable this Test until FUTURE_SEQUANA_PSA enables Memory protection
 #endif
 
-#if defined(__CORTEX_M33)
-#error [NOT_SUPPORTED] Cannot run on M33 core as SecureFault is implemented in secure-side and cant be remapped
+#if defined(__CORTEX_M23) || defined(__CORTEX_M33)
```

Comment:
I want to see this.
```suggestion
#if DOMAIN_NS == 1
```
Or anything not referencing architecture or core type, really, but this seems to be exactly the issue here - we're in the NS domain and can't get at the SecureFault. If this was improved in future so certain forms of NS domain could get at the SecureFault, it could be made more specific then.

Reply:
@kjbracey-arm @adbridge Updated as commented